server/app/services/role/RoleService.java

```diff
@@ -58,10 +64,12 @@ public Optional<CiviFormError> makeProgramAdmins(
     // Filter out CiviForm admins from the list of emails - a CiviForm admin cannot be a program
     // admin.
     ImmutableSet<String> globalAdminEmails =
-        getGlobalAdmins().stream()
```

Comment:
Not blocking: This implementation overrides the meaning of `globalAdminEmails` when the configuration value is set. I'd consider either 1) renaming it to something more generic or 2) Keeping the list the same here and updating the check on l77 to consult the variable instead.